src/L1TokenBridge.sol

```diff
@@ -84,13 +86,30 @@ contract L1TokenBridge {
         address _otherBridge,
         address _messenger
     ) {
+        _disableInitializers(); // Avoid initializing in the context of the implementation
+
         otherBridge = _otherBridge;
         messenger = CrossDomainMessengerLike(_messenger);
+    }
+
+    // --- upgradability ---
+
+    string public constant version = "1";
```

Comment:
Not feeling very strong about it, but I'd prefer to have this next to the immutables (same for the L2 bridge).

Reply:
Done